cmd/client_meta.go

```diff
@@ -44,15 +44,30 @@ func newDeleteMetaCommand() *cobra.Command {
 				return err
 			}
 			ctx := defaultContext
-			err := cdcEtcdCli.ClearAllCDCInfo(ctx)
+
+			leases, err := cdcEtcdCli.GetCaptureLeases(ctx)
+			if err != nil {
+				return errors.Trace(err)
+			}
+
+			err = cdcEtcdCli.ClearAllCDCInfo(ctx)
+			if err != nil {
+				return errors.Trace(err)
+			}
+
+			err = cdcEtcdCli.RevokeAllLeases(ctx, leases)
```

Comment:
how about revoking all leases before clear all cdc info?

Reply:
If revoking leases first, the capture will suicide, recover, and reconstruct existing changefeeds.